roles/smb_server/templates/smb.conf.j2

```diff
@@ -0,0 +1,39 @@
+#
+# smb.conf is the main Samba configuration file.
+#
+[global]
+    workgroup = WORKGROUP
+    passdb backend = tdbsam
+    map to guest = Bad User
+    usershare allow guests = No
+    server string = {{ inventory_hostname }}
+    dns proxy = no
+    log file = /var/log/samba/log.%m
+    log level = 1 passdb:2 auth:2 winbind:2
+    max log size = 1000
```

Comment:
By default the log size is 5000kb. Is there a reason for having it less than default? Do you perhaps know how fast are the log files created?

Reply:
5000 kB should be fine with the disk sizes of the new infra. I'll change that to the default...